src/zulip.rs

```diff
@@ -309,9 +309,14 @@ async fn workqueue_commands(
 
             let prs = assigned_prs
                 .iter()
-                .map(|pr| format!("- [#{pr}](https://github.com/rust-lang/rust/pull/{pr})"))
+                .map(|(pr_number, pr)| {
+                    format!(
+                        "- [#{pr_number}](https://github.com/rust-lang/rust/pull/{pr_number}) {}",
```

Comment:
Should we worry about the same "injection" attack that prompted https://github.com/rust-lang/triagebot/pull/2054 ?

Reply:
Yeah, there is probably a lot of other places where this could happen.

I'm not sure putting it in backtick would work, I often use backticks in my own PRs, which would render weird and defeat the purpose.

I feel like if we wanted to do the right we would need to bring out a markdown parser and only collect the pure text portions, but that's much more complicated. Let's defer that.